src/main/java/core/basesyntax/utiles/FigureSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.utiles;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
```

Comment:
```suggestion
public class FigureSupplier {
	private Random random = new Random();

    public Figure getRandomFigure() {
```
краще такі змінні виносити в клас щоб не створювати нові об'єкти кожного разу